Mathlib/GroupTheory/Congruence/Defs.lean

```diff
@@ -713,3 +732,62 @@ theorem induction_on_units {p : Units c.Quotient → Prop} (u : Units c.Quotient
 end Units
 
 end Con
+
+variable {S}
+
+namespace SMulCon
+
+/-- The quotient by a congruence relation preserving scalar multiplication. -/
+@[to_additive /-- The quotient by a congruence relation preserving additive action. -/]
+protected def Quotient [SMul S M] (c : SMulCon S M) : Type _ := Quotient c.toSetoid
+
+@[to_additive] instance [SMul S M] (c : SMulCon S M) : SMul S c.Quotient where
+  smul s := Quotient.map (s • ·) (@c.smul s)
+
+@[to_additive] instance [SMul S M] [One M] (c : SMulCon S M) : One c.Quotient where
+  one := ⟦1⟧
```

Comment:
Here is a question: what justifies having this instance but not a generic instance of the form "if we have a one on a type its image in any quotient is denoted 1" ?

I haven't really thought about the possible implications, so if you don't need it I would be in favor of only keeping the zero instance for `AddSMulCon`.